`make clean` breaks the build process

There is a bug somewhere in the build system. After `make clean` is executed, `make` cannot build `hyperSpec`.

## How to reproduce
I have the very same version (commit 43e439db) and all dependencies installed.
```
git clone https://github.com/cbeleites/hyperSpec.git
cd hyperSpec/
make   # works
make   # works
make clean
make   # error
```

Error message:

```
rkiselev@pc158697:~/tmp/hyperSpec$ make
update DESCRIPTION
Roxygenize
Updating hyperSpec documentation
Writing NAMESPACE
Loading hyperSpec
Loading required package: lattice
Loading required package: grid
Loading required package: ggplot2
Error in FUN(X[[i]], ...) : empty (zero-byte) input file
Calls: document ... <Anonymous> -> load_data -> unlist -> lapply -> FUN
Execution halted
Makefile:67: recipe for target 'roxygenize' failed
make: *** [roxygenize] Error 1
```

To fix this, I run
```
git clean -xfd --exclude .Rhistory --exclude .Rproj.user/ --exclude hyperSpec.Rproj
make
```
Looks like `make clean` leaves some intermediate files in the dependency chain, and `make` gets confused.

--- a/Makefile
+++ b/Makefile
@@ -10,10 +10,10 @@ clean:
 ## bootstrap target does the required processing immediately after cloning, clean, or
 ## if the installed version of hyperSpec is too old for building the vignettes
 
-bootstrap: installdeps bootstrapI chondro flu laser pkg-data | fileio-tests
+bootstrap: installdeps pkg-data bootstrapI chondro flu laser  | fileio-tests
 	@R CMD build --no-build-vignettes hyperSpec/
 	@R CMD INSTALL hyperSpec_*-$(DATE).tar.gz
-	$(MAKE) -C hyperspec/vignettes -f Makefile-local
+	$(MAKE) -C hyperSpec/vignettes -f Makefile-local
 
 bootstrapI: roxygenize
 	@R CMD build --no-build-vignettes hyperSpec/
@@ -97,7 +97,7 @@ flu:
 
 hyperspec:
 	$(MAKE) -C Vignettes/hyperspec
-	$(MAKE) -C hyperSpec/vignettes -f Makefile-local hyperspec.Rnw
+	$(MAKE) -C hyperSpec/vignettes -f Makefile-local all
 
 # laser ............................................................................................
 

--- a/Makefile
+++ b/Makefile
@@ -3,25 +3,11 @@ all: roxygenize pkg-data pkg-doc vignettes pkg-vignettes | fileio-tests
 DATE = $(shell date +%Y%m%d)
 
 clean:
-	@rm -f *~ .*~ \#*\#
-	@rm -f hyperSpec_*.tar.gz
-	@rm -rf hyperSpec.Rcheck
-	$(MAKE) -C Vignettes/baseline     clean
-	$(MAKE) -C Vignettes/chondro      clean
-	$(MAKE) -C Vignettes/fileio       clean
-	$(MAKE) -C Vignettes/flu          clean
-	$(MAKE) -C Vignettes/hyperspec clean
-	$(MAKE) -C Vignettes/laser        clean
-	$(MAKE) -C Vignettes/plotting     clean
-	$(MAKE) -C hyperSpec/inst/doc     clean
-	$(MAKE) -C hyperSpec/vignettes -f Makefile-local clean
-
-superclean:
-	@git clean -q -f -x -d
+	@git clean -q -f -x -d  --exclude .Rproj.user/ --exclude hyperSpec.Rproj
 
 # TODO: add dependency `clean`
 
-## bootstrap target does the required processing immediately after cloning, superclean, or
+## bootstrap target does the required processing immediately after cloning, clean, or
 ## if the installed version of hyperSpec is too old for building the vignettes
 
 bootstrap: installdeps bootstrapI chondro flu laser pkg-data | fileio-tests

--- a/hyperSpec/vignettes/Makefile-local
+++ b/hyperSpec/vignettes/Makefile-local
@@ -50,6 +50,9 @@ rawdata:
 
 # hyperspec vignette (aka introduction)
 
+../../Vignettes/hyperspec/strukturhyperspec.pdf:
+	@$(MAKE) -C ../../Vignettes/hyperspec strukturhyperspec.pdf
+
 hyperspec.Rnw: ../../Vignettes/hyperspec/hyperspec.Rnw ../../Vignettes/hyperspec/strukturhyperspec.pdf \
                   ../../Vignettes/hyperspec/functions.RData ../../Vignettes/hyperspec/hyperspec.bib
 	cp -av $^ ./

